src/instruments/src/Common/gauges.tsx

```diff
@@ -171,3 +173,124 @@ export const VerticalSegment: FC<VerticalSegmentProps> = ({ x, y, height, rangeS
         />
     );
 };
+
+export const splitDecimals = (value, type) => {
+    if (type === 'oil') {
+        value = value * 0.01 * 25;
+    } else if (type === 'vib') {
+        value = value < 0 ? 0.0 : value;
+    }
+    const decimalSplit = value.toFixed(1).split('.', 2);
+    return (decimalSplit);
+};
+
+export const valueRadianAngleConverter = (value, min, max, endAngle, startAngle) => {
```

Comment:
**suggestion**: use `function` here instead of an arrow function

```suggestion
export function valueRadianAngleConverter(value, min, max, endAngle, startAngle) {
```